README.md

```diff
@@ -1,7 +1,7 @@
 ### Getting started with the network-socket API ###
 
 This is a quick example of a simple HTTP client program using the
-network-socket API that is provided as a part of [mbed-os](github.com/armmbed/mbed-os).
+[network-socket API](https://os.mbed.com/docs/latest/reference/network-socket.html) that is provided as a part of [mbed-os](https://github.com/ARMmbed/mbed-os).
```

Comment:
"...[network socket API] that [Mbed OS](https://github.com/ARMmbed/mbed-os) provides."